src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [birthday, setBirthday] = useState("");
+
+  const handleSubmit = (event) => {
+    // this prevents the default behavior of the form which is to reload the entire page
+    event.preventDefault();
+
+    const data = {
+      Name: username,
+      Password: password,
+      Birthday: birthday,
```

Comment:
I think that the user object is missing the email attribute. did you also save the user document on the database without email values? If so kindly add it and make sure to add the email field in the signup form.